features/url-rewriters/cloudinary.md

```diff
@@ -32,5 +32,9 @@ The following Cloudinary transformations are supported for translation:
 | <p>image overlays</p> | <ul> <li>`l_<imagePath>` => `oi-<imagePath>`</li> </ul> | Full path to the overlaid image in the Media Library must be specified |
 | <p>named transform</p> | <ul> <li>`t_<transform_name>` => `n-<transform_name>`</li> </ul> | ImageKit does not translate the actual underlying transformations associated with your named transforms. They must be manually translated and [added in your ImageKit dashboard](/features/named-transformations.md#creating-named-transformations) while retaining the same name. |
 | <p>fl_layer_apply</p> |  | Marks the end of a group of transformations being applied in the context of an overlay. ImageKit detects the presence of an `fl_layer_apply` and applies the relevantly positioned transforms to the overlaid asset instead of the main asset. |
+| <p>start offset</p> | <ul> <li>`so_10.5599` => `so-10.56`</li> </ul> |  |
```

Comment:
this diff is messed up since you are merging into dev and dev is out of date with master. Either get dev and master synced, or create PR into master directly after addressing the rest of the comments here